### ISSUE TITLE
SwiftFormat Adding Extra Indentation to Empty Array

### ISSUE THREAD

**Issue by PompeiaPaetenari** [nicklockwood/SwiftFormat#635]
Just updated our SwiftFormat version from 0.44.5 to 0.44.12 and we encountered the following error:

Before:
```swift
JSON([
  "data": [
    "rows": [
    ],
  ],
])
```
After:
```swift
JSON([
  "data": [
    "rows": [
      ],
  ],
])
```

The only change to the config was to add: `--specifierorder convenience,override`.

As far as I can tell (as the real version of this code is much larger) is that is only happens with these empty arrays.

**Comment by nicklockwood** [comment]
@PompeiaPaetenari can you check if it also happens with 0.44.11?

**Comment by PompeiaPaetenari** [comment]
> @PompeiaPaetenari can you check if it also happens with 0.44.11?

Yes, it does it there, too.

**Comment by nicklockwood** [comment]
@PompeiaPaetenari thanks for checking. I'll get this fixed asap.

**Comment by nicklockwood** [comment]
@PompeiaPaetenari fixed in 0.44.13.
